app/controllers/application_controller.rb

```diff
@@ -98,4 +98,24 @@ def track_visit(*args)
   def suppress_hotline_link
     @suppress_contact_us_message = true
   end
+
+  def disable_analytics
```

Comment:
The only places where this is used it is used as `!@disable_anaytics`. Could it be `enable_analytics` instead (and then used as a `before_action` in the `external_users` controllers)?

Reply:
This sounds more readable to me. I am happy to make these changes for Pete while he is off and give it a test

Reply:
I could be reading this wrong, but isn't it used in `statistics_controller.rb` and `widgets_controller.rb`?

Reply:
The `before_action` calls this method and so sets the variable. If it is changed to `enable_analytics` then the `before_action`s in each of the controllers will need to be changed so that it is set to `true` in the relevant cases.

Reply:
Actually, I'm having second thoughts about this. The default (as it stands) is for analytics to be enabled and this is the case for all controllers, not just the five that are being changed here, so an `enable_analytics` action would have to be applied to all. It makes more sense for the setting to be 'disable'.

I'm currently looking at the `GoogleAnalytics::DataTracking` module to see if that is where this lives.

Reply:
I've created a branch where I've put this flag inside `GoogleAnalytics::DataTracker`:

https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/compare/1695-layout-page-migration...1695-layout-page-migration_disable_analytics

I didn't push it straight into this PR as I'm not totally familiar with how `GoogleAnalytics::DataTracking`, so this PR can go as it is and then that change can be reviewed later.